mkdocs.yml

```diff
@@ -8,6 +8,7 @@ edit_uri: ""
 nav:
     - Home: index.md
     - Getting Started: getting_started.md
+    - Presidio Revamp (V2): presidio_V2.md
```

Comment:
Do we want this on the 1st level nav? Maybe under API or development?

Reply:
moved under API reference, not sure if that is the right place though